lib/mint/web_socket/frame.ex

```diff
@@ -21,6 +21,34 @@ defmodule Mint.WebSocket.Frame do
   defrecord :ping, shared
   defrecord :pong, shared
 
+  @type continuation_frame() ::
+          record(:continuation, reserved: binary(), mask: binary, data: binary, fin?: boolean)
```

Comment:
`reserved` should be a bitstring since it's only ever 3 bits, as long as the dialyzer accepts that